ROMFS/px4fmu_common/init.d/rc.mavlink

```diff
@@ -0,0 +1,233 @@
+#!nsh
+# MAVLINK start script.
+#
+# NOTE: environment variable references:
+#    If the dollar sign ('$') is followed by a left bracket ('{') then the
+#    variable name is terminated with the right bracket character ('}').
+#    Otherwise, the variable name goes to the end of the argument.
+#
+#
+# NOTE: COMMENT LINES ARE REMOVED BEFORE STORED IN ROMFS.
+#
+# UART mapping on FMUv2/3/4:
+#
+# UART1			/dev/ttyS0		IO debug (except v4, there ttyS0 is the wifi)
+# USART2		/dev/ttyS1		TELEM1 (flow control)
+# USART3		/dev/ttyS2		TELEM2 (flow control)
+# UART4
+# UART7							CONSOLE
+# UART8							SERIAL4
+#
+#
+# UART mapping on FMUv5:
+#
+# UART1			/dev/ttyS0		GPS
+# USART2		/dev/ttyS1		TELEM1 (flow control)
+# USART3		/dev/ttyS2		TELEM2 (flow control)
+# UART4			/dev/ttyS3		TELEM4
+# USART6		/dev/ttyS4		TELEM3 (flow control)
+# UART7			/dev/ttyS5		?
+# UART8			/dev/ttyS6		CONSOLE
+
+
+#
+# Set default values
+#
+set MAVLINK_F default
+set MAVLINK_COMPANION_DEVICE_1 /dev/ttyS1
```

Comment:
Let's think of a better name for this if we're going to go in this direction. It goes beyond both mavlink and companion devices to become generic serial port configuration.

Reply:
Also agreed.  I'll fix it up asap. Thanks!